src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

```diff
@@ -12,6 +12,12 @@ Notes](../../RELEASENOTES.md).
   exports are correctly marked as successful.
   ([#6099](https://github.com/open-telemetry/opentelemetry-dotnet/pull/6099))
 
+* Fixed an issue where `Status.Description` exceeding 127 bytes resulted in
+  incorrect length encoding, leading to serialization errors. The length
+  encoding now correctly handles descriptions longer than 127 bytes, ensuring
+  accurate serialization and preventing related failures.
+  ([#](https://github.com/open-telemetry/opentelemetry-dotnet/pull/))
```

Comment:
```suggestion
* Fixed an issues causing trace exports to fail when
  `Activity.StatusDescription` exceeds 127 bytes.
  ([#6119](https://github.com/open-telemetry/opentelemetry-dotnet/pull/6119))
```

Wanted to make it clear it impacts tracing but also simplified a bit too. Just a suggestion feel free to take or not.